### DISCUSSION TITLE
WWAData の初期化場所を統一、または何かしらの規則をつくる

### DISCUSSION THREAD

**Discussion by matsuyuki-a** [WWAWing/WWAWing#552] [Ideas]
https://github.com/WWAWing/WWAWing/pull/303#discussion_r519410845

WWAData の項目ごとに初期化している場所が異なるので、統一するか、初期化場所について何かしらのルールを明確化する

**Comment by matsuyuki-a** [comment]
リポジトリ分割 (#226 )までにやらないと、LoaderとEngineが別リポジトリになるので、初期化まわりがちょっと面倒なことになりそう

**Comment by aokashi** [comment]
WWA のマップデータの定義は WWA 共通リポジトリに、 WWA のセーブデータの定義は WWA Wing 本体に置けないかな
手間かかりそうだけど
